pkg/asset/cluster/tfvars.go

```diff
@@ -39,25 +40,55 @@ func (t *TerraformVariables) Dependencies() []asset.Asset {
 		new(rhcos.Image),
 		&bootstrap.Bootstrap{},
 		&machine.Master{},
+		&machines.Master{},
 	}
 }
 
 // Generate generates the terraform.tfvars file.
 func (t *TerraformVariables) Generate(parents asset.Parents) error {
 	clusterID := &installconfig.ClusterID{}
 	installConfig := &installconfig.InstallConfig{}
-	bootstrap := &bootstrap.Bootstrap{}
-	master := &machine.Master{}
+	bootstrapIgnAsset := &bootstrap.Bootstrap{}
```

Comment:
the other variables arounf these don't add the `asset` suffix, so maybe just `bootstrapIgn` ?

Reply:
> the other variables arounf these don't add the `asset` suffix, so maybe just `bootstrapIgn` ?

I was saving `bootstraIgn` for [here][1].

[1]: https://github.com/openshift/installer/pull/792/files#diff-bfad726be2a317f0c2bee99d4d62e23eR57

Reply:
looks like `boostrapIgn` is only used once [here](https://github.com/openshift/installer/pull/792/commits/c1c86862ff109bde9062bae135684238750432da#diff-bfad726be2a317f0c2bee99d4d62e23eR84)

why create a new variable for one time use....

Reply:
> looks like boostrapIgn is only used once here
>
> why create a new variable for one time use....

Because [that's what we do in master][1]?  But if you don't like our current local variables, I can remove the single-use ones in this PR.  If you want that, did you just want me to clean them out of this function?

[1]: https://github.com/openshift/installer/blob/da6d45b36c8b39e40fe28a31e53337405e28162d/pkg/asset/cluster/tfvars.go#L54